cves/CVE-2015-8363.yml

```diff
@@ -199,8 +206,10 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    The vulnerability occured via an exploit in the JPEG 2000 spec 
+    by having multiple size markers.
```

Comment:
What are markers? This probably could be answered by explaining more about what JPEG 2000 is